google-cloud-storage/test/google/cloud/storage/retry/utils.rb

```diff
@@ -0,0 +1,464 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "securerandom"
+
+module MethodMapping
+
+  CONF_TEST_SERVICE_ACCOUNT_EMAIL = "my-service-account@test.iam.gserviceaccount.com"
+  CONF_TEST_ACL_ENTITY = "my-entity@example.net"
+  CONF_TEST_PUBSUB_TOPIC_NAME = "yet-another-topic"
+  CONF_TEST_FILE_CONTENT = "my-test-file"
+  CONF_TEST_FILE_PATH = "my-test-file.txt"
+
+  ########################################################################################################################################
+  ### Method Invocation Mapping ##########################################################################################################
+  ########################################################################################################################################
+
+  # Method invocation mapping is a map whose keys are a string describing a standard
+  # API call (e.g. storage.objects.get) and values are a list of functions which
+  # wrap library methods that implement these calls. There may be multiple values
+  # because multiple library methods may use the same call (e.g. get could be a
+  # read or just a metadata get).
+
+  def self.get
+  {
+    "storage.bucket_acl.list" => [:list_bucket_acls],
```

Comment:
Nit: could you sort these by resource / alphabetically? Otherwise it's a bit difficult to see all of what's implemented.

Reply:
Makes sense. Updated it alphabetically.